buildconfig/Jenkins/buildscript

```diff
@@ -10,18 +10,45 @@
 # PARAVIEW_DIR should be set in the configuration of each slave.
 ###############################################################################
 SCRIPT_DIR=$(dirname "$0")
+XVFB_SERVER_NUM=101
+ULIMIT_CORE_ORIG=$(ulimit -c)
+
+if [ $(command -v xvfb-run) ]; then
+  X11_RUNNER="xvfb-run"
+  X11_RUNNER_ARGS1=--server-args="-screen 0 640x480x24"
+  X11_RUNNER_ARGS2=--server-num=${XVFB_SERVER_NUM}
+else
+  X11_RUNNER="eval"
+  X11_RUNNER_ARGS1=""
+  X11_RUNNER_ARGS2=""
+fi
+
+###############################################################################
+# Functions
+###############################################################################
+
+function onexit {
+  if [[ ${USE_CORE_DUMPS} == true ]]; then
+    ulimit -c $ULIMIT_CORE_ORIG
+  fi
+}
+
+function run_with_xvfb {
```

Comment:
Could the `command -v xvfb-run` check just be done in this function? This would get around the additional variables for parameters, etc.